doc/source/user/quickstart.rst

```diff
@@ -1112,8 +1112,8 @@ indices for each dimension must have the same shape.
            [[10,  9],
             [11, 11]]])
 
-Naturally, we can put ``i`` and ``j`` in a sequence (say a list) and
-then do the indexing with the list.
+In Python, ``arr[i, j]`` is exactly the same as ``arr[(i, j)]``---so naturally, we can put ``i`` and ``j`` in a ``tuple`` and
```

Comment:
Grammatical nit-pick - I prefer index as a verb.

Also, standalone pronouns (e.g. "that") are often ambiguous, though I think it's pretty clear in this case.

> we can put ``i`` and ``j`` in a tuple and then do indexing with that

could become something like:
"we can construct a tuple containing ``i`` and ``j``, and index the array with the constructed tuple."

Just my two cents, feel free to ignore.

Reply:
Note the current wording is consistent with the wording below,

> However, we can not do this by putting ``i`` and ``j`` into an array,

Let's keep them consistent, either by leaving things as is, or by changing both. I'm ambivalent, this PR is already a clear improvement over master.